modules/javafx.graphics/src/main/java/javafx/scene/text/TextFlow.java

```diff
@@ -244,6 +248,18 @@ public final PathElement[] underlineShape(int start, int end) {
         return getRange(start, end, TextLayout.TYPE_UNDERLINE);
     }
 
+    /**
+     * Returns the shape for the strike-through in local coordinates.
+     *
+     * @param start the beginning character index for the range
+     * @param end the end character index (non-inclusive) for the range
+     * @return an array of {@code PathElement} which can be used to create a {@code Shape}
+     * @since 25
+     */
+    public final PathElement[] strikeThroughShape(int start, int end) {
```

Comment:
Perhaps this method should be renamed to `getStrikeThroughShape()` to be consistent with the proposed evolution of the buggy methods in https://bugs.openjdk.org/browse/JDK-8341438